src/repeated_game.jl

```diff
@@ -438,3 +440,118 @@ function outerapproximation(rpd::RepGame2; nH=32, tol=1e-8, maxiter=500,
 
     return vertices
 end
+
+
+function AS{T}(rpd::RepeatedGame{2, T}; max_iter::Integer=1000,
+               plib=getlibraryfor(2, T), tol::Float64=1e-5, u=nothing)
+
+    # Initialize W0 with each entries of payoff bimatrix
+    v_old = _payoff_points(rpd.sg)
+
+    if u == nothing
+        u = [minimum(rpd.sg.players[1].payoff_array),
+             minimum(rpd.sg.players[2].payoff_array)]
+    end
+
+    # create VRepresentation and Polyhedron
+    V = SimpleVRepresentation(v_old)
```

Comment:
This part is also what I am really concerned about. What I am trying to do here is to use a Matrix `v_new` to create `VRepresentation` without any redundant points (because the computation of intersection of two polyhedra is really slow if there are many redundant points).

I am not sure why `Polyhedra.jl` do not implement [`removevredundancy!(p::VRep)`](https://github.com/JuliaPolyhedra/Polyhedra.jl/blob/master/src/redundancy.jl#L12), and I have to call the internal function `removevredundancy(vrep::VRep, hrep::HRep; strongly=true)` to remove the redundant points, which I don't know if it's efficient or not. I think we need to look into `Polyhedra.jl` deeply to see if I can improve this (very important) part.